cmd/env/pull/pull.go

```diff
@@ -58,6 +58,49 @@ func RunPull(args []string, forceFlag bool) error {
 		return err
 	}
 
+	// Check if this is a monorepo
+	if manifest.IsMonorepoProject() && manifest.Project != nil {
+		// Store current directory
+		currentDir, err := os.Getwd()
+		if err != nil {
+			return fmt.Errorf("failed to get current directory: %w", err)
+		}
+
+		// Pull for each workspace member
```

Comment:
`member` is a word that has other meanings within the Hyphen system. Could you just refer to them as `apps` here? Isn't that what we're actually interacting with in APIx?